polaris-react/src/components/IndexTable/IndexTable.scss

```diff
@@ -216,7 +216,19 @@ $loading-panel-height: 53px;
 
       #{$se23} & {
         // Bump specificity
-        background-color: var(--p-color-bg-interactive-selected);
+        background-color: var(--p-color-bg-subdued-active);
+      }
+    }
+  }
+
+  // stylelint-disable-next-line no-duplicate-selectors -- se23 temporary styles
+  #{$se23} & {
+    &.TableRow-hovered {
+      // stylelint-disable-next-line selector-max-class, selector-max-combinators, selector-max-compound-selectors -- se23 temporary styles
+      &,
+      .TableCell-first,
+      .TableCell-first + .TableCell {
+        background-color: var(--p-color-bg-subdued-hover);
```

Comment:
Can this be moved up to [line 199](https://github.com/Shopify/polaris/pull/9442/files#diff-e9d0c2a006c030f315115b92e46c3b44667bef0b5e909e1b3af852e6eb2b605bR199) (instead of duplicating all the selectors)?

Reply:
I deliberately moved it down in the cascade because otherwise the selected state overrides the hover state and I think we want it the other way around.